src/Common/Common.Api/Workspaces/Workspace.cs

```diff
@@ -61,5 +61,15 @@ public class Workspace
                 Users = workspace.Users?.Select(x => (GroupUserAccessRight)x).ToList()
             };
         }
+
+        public bool IsOrphanedWorkspace()
```

Comment:
Should be a property instead of a method, drop Workspace from name as the type is Workspace

Reply:
You need to update Microsoft.PowerBI.Commands.Workspaces.format.ps1xml with the new property otherwise it won't show to the user